src/model/FeatureCol.js

```diff
@@ -38,7 +39,9 @@ const FeatureCol = Collection.extend({
   // tries to auto-fit the rows
   // not a very efficient algorithm
   assignRows: function() {
```

Comment:
For context, `assignRows` is an algorithm that'll figure out the vertical placing of features such that there's minimal no:of rows required to pack features together.